### PR TITLE
Bugfix/pp9 10517 handle slow networks

### PR DESCRIPTION
**Pull request by rjs-picturepark**

### CODE REVIEW

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/basket/basket.component.html [hunk 1]
```diff
@@ -24,8 +24,8 @@
       </div>
     </mat-toolbar>
     <div class="basket__items">
-      <div class="basket__item-container" *ngFor="let basketItem of basketItems; trackBy: trackByBasket">
-        <pp-basket-item class="basket__item" [item]="basketItem" (dblclick)="previewItem(basketItem)"></pp-basket-item>
+      <div class="basket__item-container" *ngFor="let basketItem of basketItems; trackBy: basketItemTracker">
```
Comment (rjs-picturepark) [inline src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/basket/basket.component.html:27]: revert naming to old convention

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/shared-module/components/content-item-thumbnail/content-item-thumbnail.component.html [hunk 2]
```diff
@@ -1,23 +1,24 @@
...
-      [ngClass]="{ shadow: shadow }"
-      class="content-item-thumbnail__image"
-      [src]="url"
-      [alt]="'ContentBrowser.NoImageForItem' | pptranslate"
-      (error)="updateUrl($event)"
-    />
-  </ng-container>
-</div>
-<ng-template #virtualContent>
+<ng-container *ngIf="virtualItemHtml; else virtualContent">
```
Comment (rjs-picturepark) [inline src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/shared-module/components/content-item-thumbnail/content-item-thumbnail.component.html:1]: the else has the wrong name

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/shared-module/components/content-item-thumbnail/content-item-thumbnail.component.ts [hunk 3]
```diff
@@ -3,49 +3,43 @@ import {
...
-  @Input() shareItem: ShareDetail;
+  @Input() shareItem?: ShareDetail;
 
   /**
    * If true the image will have a shadow box around
    */
   @Input() shadow: boolean;
 
-  public isLoading = false;
+  public isLoading = true;
```
Comment (rjs-picturepark) [inline src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/shared-module/components/content-item-thumbnail/content-item-thumbnail.component.ts:42]: Set on loading only when necessary

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/shared-module/services/basket/basket.service.ts [hunk 4]
```diff
@@ -1,70 +1,110 @@
 import { Injectable } from '@angular/core';
 import { BehaviorSubject, Observable } from 'rxjs';
+import { Content, ContentService, fetchContents } from '@picturepark/sdk-v1-angular';
+import * as _ from 'lodash';
```
Comment (rjs-picturepark) [inline src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/shared-module/services/basket/basket.service.ts:4]: Get rid of lodash

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/basket/basket.component.html [hunk 5]
```diff
@@ -25,7 +25,7 @@
     </mat-toolbar>
     <div class="basket__items">
       <div class="basket__item-container" *ngFor="let basketItem of basketItems; trackBy: trackByBasket">
-        <pp-basket-item class="basket__item" [item]="basketItem" (dblclick)="previewItem(basketItem)"></pp-basket-item>
+        <pp-basket-item class="basket__item" [item]="basketItem" (dblclick)="previewItem(basketItem)"> </pp-basket-item>
```
Comment (tc4rdpp) [inline src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/basket/basket.component.html:28]: was this added by Prettier?

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/shared-module/services/basket/interfaces/basket-change.ts [hunk 6]
```diff
@@ -0,0 +1,10 @@
+export interface BasketChange {
```
Comment (tc4rdpp) [inline src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/shared-module/services/basket/interfaces/basket-change.ts:1]: I would move this into an interfaces folder directly inside the shared-modules

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/shared-module/components/content-item-thumbnail/content-item-thumbnail.component.ts [hunk 7]
```diff
@@ -75,6 +76,7 @@ export class ContentItemThumbnailComponent extends BaseBrowserItemComponent<Cont
         } else {
           this.thumbnailUrl$ = this.loadItem.pipe(
             switchMap(() => {
+              this.isLoading = true;
```
Comment (rjs-picturepark) [inline src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/shared-module/components/content-item-thumbnail/content-item-thumbnail.component.ts:79]: Change to a tap function

**Pull request merged**